src/unity/lib/unity_sframe.cpp

```diff
@@ -1950,6 +1950,186 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
       }
     };
 
+    auto getAccordian = [self, &reader, &ew, &column_names, &empty_tz](std::string column_name, size_t index) {
```

Comment:
Accordion (with an o). And I'm not sure I'd want to use the name of the specific UI control here - can we come up with a name that represents the information being passed through, rather than the way it's displayed to the user? How about `getFlexibleTypeValue`?